admin_guide/image_signatures.adoc

```diff
@@ -53,27 +54,113 @@ The `atomic` command line interface (CLI), version 1.12.5 or greater, provides
+It is pre-installed on RHEL Atomic Host systems. For installing the *atomic*
+package on a RHEL host, see
+xref:../../install_config/install/host_preparation.adoc#enabling-image-signature-support[Enabling Image Signature Support].
+endif::[]
+
+[IMPORTANT]
+====
+The `atomic` CLI uses the authenticated credentials from `oc login`. Be sure to
+use the same user on the same host for both `atomic` and `oc` commands. For example,
+if you execute `atomic` CLI as `sudo`, be sure to log in to {product-title}
+using `sudo oc login`.
+====
 
-You can sign an image at push time:
+In order to attach the signature to the image, the user must have the
+`image-signer` cluster role.
+ifdef::openshift-origin,openshift-enterprise[]
+Cluster administrators can add this using:
 
 ----
-$ atomic push [--sign-by ...] <image>
+$ oc adm policy add-cluster-role-to-user system:image-signer <user_name>
 ----
+endif::[]
 
-Or you can sign an image that has already been pushed to the registry:
+Images may be signed at push time:
 
 ----
-$ atomic sign [--sign-by ...] <image>
+$ atomic push [--sign-by <gpg_key_id>] --type atomic <image>
 ----
 
+Signatures are stored in {product-title} when the **atomic** transport type
+argument is specified. See xref:../security/deployment.adoc#security-deployment-signature-transports[signature transports]
+for more information.
+
 For full details on how to set up and perform image signing using the `atomic`
 CLI, see the
-link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/signing_container_images[RHEL Atomic Host Managing Containers: Signing Container Images] documentation.
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/signing_container_images[RHEL Atomic Host Managing Containers: Signing Container Images] documentation
+or the `atomic push --help` output for argument details.
 
 A specific example workflow of working with the `atomic` CLI and an OpenShift
 Container Registry is documented in the
 link:https://access.redhat.com/articles/2750891#working-with-openshift-and-atomic-registry[Container Image Signing Integration Guide].
 
+[[verifying-image-signatures-using-openshift-cli]]
+== Verifying Image Signatures Using OpenShift CLI
+
+You can verify the signatures of an image imported to an OpenShift Container
+Registry using the `oc adm verify-image-signature` command. This command
+verifies if the image identity contained in the image signature can be trusted
+by using the public GPG key to verify the signature itself then match the
+provided expected identity with the identity (the pull spec) of the given image.
+
+By default, this command uses the public GPG keyring located in
+*_$GNUPGHOME/pubring.gpg_*, typically in path *_~/.gnupg_*. By default, this
+command does not save the result of the verification back to the image object.
+To do so, you must specify the `--save` flag.
```

Comment:
I see --save in the command below. Should you add a specify the `--save` flag, as shown below?